daemon/emer-persistent-cache.c

```diff
@@ -778,8 +778,11 @@ emer_persistent_cache_finalize (GObject *object)
       g_error_free (error);
```

Comment:
In the last commit message did you mean a "persistent" cache rather than a "permanent" cache?